src/containers/UserSettings/Setting.tsx

```diff
@@ -0,0 +1,84 @@
+import type {ReactNode} from 'react';
+
+import {RadioButton, Switch} from '@gravity-ui/uikit';
+import {Settings} from '@gravity-ui/navigation';
+
+import {LabelWithPopover} from '../../components/LabelWithPopover/LabelWithPopover';
+
+import {useSetting} from '../../utils/hooks';
+
+import {b} from './UserSettings';
+
+export type SettingsElementType = 'switch' | 'radio';
+
+export interface SettingProps {
+    type?: SettingsElementType;
+    title: string;
+    settingKey: string;
+    withHelpPopover?: boolean;
+    helpPopoverContent?: ReactNode;
+    values?: {value: string; content: string}[];
+    defaultValue?: unknown;
+}
+
+export const Setting = ({
+    type = 'switch',
+    settingKey,
+    title,
+    withHelpPopover,
```

Comment:
It seems to be redundant. You may just check if `helpPopoverContent` is presented.